src/workerd/api/global-scope.c++

```diff
@@ -94,6 +94,14 @@ void ExecutionContext::abort(jsg::Lock& js, jsg::Optional<jsg::Value> reason) {
   js.terminateExecutionNow();
 }
 
+void ExecutionContext::addEventListener(
+    kj::String type, jsg::Function<void(jsg::Ref<api::Event>)> handler) {
```

Comment:
Likewise here... feels incomplete if it doesn't match the behavior of `EventTarget`.

Reply:
Let's move this to the first comment for further discussion.